code/src/components/Message.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+const Message = ({ loading, newMessage, onNewMessageChange, onFormSubmit }) => {
+
+    // SHOWN WHEN IMAGE RELOADS WHICH I HAVENT MANAGED TO FIX, DO NOT WANT IT TO SHOW..
+
+    if (loading) {
+        return <div className='loading-container'>
+            <img className='loading' src='./icons/heart.png' alt="heart icon" />
+        </div>
```

Comment:
Not sure I completely follow, but a little insight: the way your code is structured, I believe the entire page renders again every time a message is submitted. Assuming that's correct, you don't currently have a way of avoiding showing the loading heart when a message is submitted. It will happen both when you first load the page and when you submit because essentially the page starts all over again by reloading everything.